src/sentry_core.c

```diff
@@ -934,6 +934,8 @@ sentry_transaction_finish(sentry_transaction_t *opaque_tx)
     sentry_value_t trace_context
         = sentry__value_get_trace_context(opaque_tx->inner);
     sentry_value_t contexts = sentry_value_new_object();
+    sentry_value_set_by_key(
+        contexts, "data", sentry_value_get_by_key(tx, "extra"));
```

Comment:
could this cause issues in tests where we didn't expect the transaction's data to be passed through before (e.g. context data differs because it's expected to be empty?)

Reply:
well, I hope it at least misses the "extra" key.